docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -141,6 +141,11 @@ on the same platform.
     - If `SGX_AESM_ADDR` is defined (regardless of its value), out-of-process
     quote generation is done.
     - Otherwise if it is not defined, in-process quote generation is done.
+    - `libsgx-quote-ex` has to be uninstalled and `SGX_AESM_ADDR` has to be unset in order to run in-process quote generation on Linux.
```

Comment:
This added text is not accurate and feels confusing. In-process quote generation is done if quote-ex is uninstalled or SGX_AESM_ADDR is unset.

I think it's better to drop the added text here, and update the next section "Priority between ..." as below:

An SGX platform can have either the DCAP library or the quote-ex library,
or both of them installed.
* When only the DCAP library is installed, it will be used, and only in-process quote generation is supported.
    * In this case, `SGX_AESM_ADDR` must be unset. Otherwise the DCAP library will return error.
* When only the quote-ex library is installed, it will be used, and only out-of-process quote generation is supported.
    * In this case, `SGX_AESM_ADDR` must be set. Otherwise, the OE SDK will try to load the unavailable DCAP library and will fail.
* When both libraries are installed:
    * If `SGX_AESM_ADDR` is not set, the DCAP library is used and is configured for in-process quote generation. This preserves the default behavior for the OE SDK before the integration of the quote-ex library.
    * Otherwise if `SGX_AESM_ADDR is set, the quote-ex library will be used, as it supports more evidence formats.

On Linux platforms, to be able to use the DCAP library for in-process quote generation, the user running the process needs to be added to the `sgx_prv` group, with the following command:
- `sudo usermod -a -G sgx_prv <username>`